Directory.Packages.props

```diff
@@ -34,6 +34,7 @@
     <PackageVersion Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="$(MicrosoftCodeAnalysisBannedApiAnalyzersVersion)" />
     <PackageVersion Include="Microsoft.CodeAnalysis.PublicApiAnalyzers" Version="$(MicrosoftCodeAnalysisPublicApiAnalyzersVersion)" />
     <PackageVersion Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.14.15" />
+    <PackageVersion Include="Moq.Analyzers" Version="0.3.0" />
```

Comment:
Note: This package is BSD-3-Clause license. I'm not super familiar with that but thought it's worth noting.

Reply:
I have seen other projects using it so we should be fine. The only question is whether we need to specify that in the third-party licenses given this is only for testing and not shipped.